OpenRA.Mods.RA/Traits/Disguise.cs

```diff
@@ -115,7 +115,14 @@ void DisguiseAs(Actor self, Actor target)
 				var tooltip = target.TraitsImplementing<IToolTip>().FirstOrDefault();
 				AsTooltipInfo = tooltip.TooltipInfo;
 				AsPlayer = tooltip.Owner;
-				AsSprite = target.Trait<RenderSprites>().GetImage(target);
+
+				// Take the image of the target's disguise, if it exist.
+				// E.g., SpyA is disguised as a dog, SpyB then targets SpyA: We should use the dog image.
```

Comment:
Probably, I don't get why it's a pain to update it to something that the mod can actually do.